app/models/mixins/ownership_mixin.rb

```diff
@@ -11,9 +11,27 @@ module OwnershipMixin
     virtual_column :owned_by_current_user,                :type => :boolean,    :uses => :evm_owner_userid
     virtual_column :owning_ldap_group,                    :type => :string,     :uses => :miq_group
     virtual_column :owned_by_current_ldap_group,          :type => :boolean,    :uses => :owning_ldap_group
+
+    # Determine whether to return vms ownded by the current user or not
+    # Requires `with_miq_group` scope to function
+    #
+    # Resulting SQL: "(LOWER(miq_groups.description) = 'some_miq_group'"
+    #
+    # Will result in the following when used with MiqExpression:
+    #
+    #   WHERE ((LOWER("miq_groups"."description") = 'some_miq_group') = 'true')
+    virtual_attribute :owned_by_current_ldap_group, :boolean, :arel => (lambda do |t|
```

Comment:
Is there no associated method?  (in other words, I didn't expect to see a bare virtual_attribute, but instead a virtual_column...is there a difference?)